pkg/resource/deploy/step_generator.go

```diff
@@ -67,6 +67,8 @@ type stepGenerator struct {
 
 	// a map from old names (aliased URNs) to the new URN that aliased to them.
 	aliased map[resource.URN]resource.URN
+	// a list of aliases for each resource discovered so far
+	aliases map[resource.URN]resource.URN
```

Comment:
This is different from `aliased` above? How? Also "a list of aliases for each resource discovered so fa" should this be map[resource.URN][]resource.URN`?

Reply:
Ah!  Sorry, the comment is incorrect (I can fix it) - it's actually the inverse mapping of `aliased` above - that's a map from old names to things that aliased to them.  `aliases` is a map from _new_ names to the aliases those names used.

Reply:
Ah nice! Yes a comment relating the two will help the reader for sure.